src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,12 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/Types';
+
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
+    {user?.name}
```

Comment:
so you can remove it
```suggestion
  user: User;
};

export const UserInfo: React.FC<Props> = ({ user }) => (
  <a className="UserInfo" href={`mailto:${user.email}`}>
    {user.name}
```

Reply:
exactly! I remowed that operator